e2e/util/timeout.go

```diff
@@ -9,8 +9,20 @@ import (
 )
 
 const (
-	Timeout       = 600 * time.Second
+	Timeout       = 10 * time.Minute
 	RetryInterval = 5 * time.Second
+
+	// UnprotectTimeout defines the maximum time allowed for resource deletion during unprotect.
+	// This includes waiting for DRPCs and related resources to be fully removed.
+	UnprotectTimeout = 10 * time.Minute
+
+	// UndeployTimeout defines the maximum time allowed for all resource deletions during undeploy.
+	// TODO: Undeploy usually takes about 10 seconds, we keep it longer because we call it in
+	// parallel with unprotect in ramenctl during cleanup and it needs more testing.
+	UndeployTimeout = 3 * time.Minute
```

Comment:
I think we will have to move this to the config, so ramenctl can change the timeouts when running unprotect and undeploy in parallel, but adding separate timeout per dr action makes sense and it a small step towards better timeouts.

Reply:
ack, added to config in a separate commit.